### PR TITLE
Old stuff had been removed from the update routine

### PR DESCRIPTION
**Pull request by annando**
Like discussed some time ago, I removed much old stuff from the update routine. If people now want to update from a very old version, they have prior to update the 3.5.4, then they can continue. Of course we throw out an error if someone tries it anyway.

I don't see this split update process as a big problem, but this prevents us from loading unneeded stuff with every page request.

### CODE REVIEW

File: update.php [hunk 1]
```diff
@@ -13,1607 +11,27 @@
...
- * At the top of the file "boot.php" is a define for DB_UPDATE_VERSION. Any time there is a change
- * to the database schema or one which requires an upgrade path from the existing application,
- * the DB_UPDATE_VERSION and the UPDATE_VERSION at the top of this file are incremented.
+ * Database structure changes are done in src/Database/DBStructure.php
  *
- * The current DB_UPDATE_VERSION is stored in the config area of the database. If the application starts up
- * and DB_UPDATE_VERSION is greater than the last stored build number, we will process every update function
- * in order from the currently stored value to the new DB_UPDATE_VERSION. This is expected to bring the system
- * up to current without requiring re-installation or manual intervention.
+ * If there is a need for a post procession to a structure change, update this file
```
Comment (MrPetovan) [inline update.php]: “post process”

File: update.php [hunk 2]
```diff
@@ -13,1607 +11,27 @@
...
+ * numbered script in this file.
  *
- * If you change the database schema, the following are required:
- *    1. Update the file src/Database/DBStructure.php to match the new schema.
- *    2. If there is a need for a post procession, update this file by adding a new function at the end with the number of the current DB_UPDATE_VERSION.
- *       This function should perform some post procession steps but no database updates.
- *    3. Increment the DB_UPDATE_VERSION in boot.php *AND* the UPDATE_VERSION in this file to match it
- *    4. TEST the upgrade prior to checkin and filing a pull request.
+ * Example:
+ * You are currently on version 4711 and you are preparing changes that demand an update script.
```
Comment (MrPetovan) [inline update.php]: Wait, this is counter-intuitive.

If I add a database structure change and want to apply a post-update change, the I expect the update function to be numbered exactly the same number as the one I changed myself when I updated DBStructure.php

Comment (annando) [reply update.php]: I only have documented the current behaviour, I haven't invented it. :-)

Comment (annando) [reply update.php]: Okay, I changed it.

**Pull request merged**